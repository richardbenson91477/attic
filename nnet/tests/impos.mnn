1 3 1
0.01 0.1 0.1 100.0 10000 10
impos.mni
