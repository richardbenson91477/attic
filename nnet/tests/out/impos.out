0.914443 
-0.012443 
-0.900110 
0.016337 
0.913809 
